javascript/first-js-oop-project.md

```diff
@@ -0,0 +1,92 @@
+# Short mini project to prepare apprentices for their first JS OOP project
+
+### Projected Time
+
+Total: 65 min- 80min
+- Lesson: 20 min
+- Guided Practice: 15 min
+- Independent Practice: 15 min
+- Materials: 20 min
+
+### Prerequisites
+
+- [Java script lesson 7](/javascript/javascript-7.md)
```

Comment:
(/javascript/javascript-7.md) -> (../javascript/javascript-7.md)

Reply:
why those ..?

Reply:
Those are needed...

Reply:
ohk. Done.

Reply:
for relative reference to a repository file